src/index.html

```diff
@@ -8,34 +8,46 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <div class="card" data-qa="card">
+    <div class="wrapper">
```

Comment:
[IMPORTANT]

6. [BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.

no "wrapper" block or elements related to it necessary